.github/workflows/github-pages.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Setup mdBook
-        uses: peaceiris/actions-mdbook@v1
+        uses: peaceiris/actions-mdbook@v2
```

Comment:
LGTM. It doesn't look like we specify an explicit Node version in this file which seems like the only breaking change in https://github.com/peaceiris/actions-mdbook/blob/v2.0.0/CHANGELOG.md